libra_service.js

```diff
@@ -188,14 +188,14 @@ class Libra {
     await streamWrite(writable, `query account_state ${this.userAddress}\n`)
     await sleep(1000)
     if (event === 'sent') {
-      console.log('sent_events_count', this.sent_events_count)
       if (this.sent_events_count > 0) {
         console.log(`query event ${this.userAddress} ${event} ${this.sent_events_count - 1} false 5\n`)
         await streamWrite(writable, `query event ${this.userAddress} ${event} ${this.sent_events_count - 1} false 5\n`)
         await sleep(1000)
       }
     } else {
       if (this.received_events_count > 0) {
+        console.log(`query event ${this.userAddress} ${event} ${this.received_events_count - 1} false 10\n`)
```

Comment:
Could you please make this command as const, and reuse it on streamWrite.

Reply:
And perhaps the line above on "if" in the same way as well, thanks.

Reply:
ok, I got it.